packages/sit-onyx/src/components/OnyxSelectInput/OnyxSelectInput.vue

```diff
@@ -29,7 +29,7 @@ const emit = defineEmits<{
   /**
    * Emitted when the select input is clicked (and is not disabled).
    */
-  click: [];
+  selectInputClick: [];
```

Comment:
What do you think about just `inputClick`? Since it should be clear that this component is the select :)